PlaceRouteHierFlow/placer/ConstGraph.cpp

```diff
@@ -4562,6 +4562,26 @@ void ConstGraph::UpdateTerminalinHierNode(design& caseNL, PnRDB::hierNode& node)
       temp_pin.name = node.Terminals.at(i).name;
       temp_pin.type = node.Terminals.at(i).type;
       
+      /*
+      //added by yaguang - 10/28/2020
```

Comment:
Remove commented out code?

Reply:
@soneryaldiz 

Yes. You can either keep it or remove it, since this part code is already commented.